backend/server.js

```diff
@@ -2,15 +2,14 @@ const express = require("express");
 const getBreadPun = require("./getBreadPun");
 
 const app = express();
-
-app.get("/api/breadpuns", (req, res) => {
-  res.json(getBreadPun());
-});
-
```

Comment:
Why would you put the .get above where the server is even started

Reply:
Shouldn’t matter since it’s async but it’s general convention to put the listen method at the end so she probably got it from the Express docs. :)

Reply:
app.get registers routes instead of instantiating them which is why the convention is to configure your server (including registering routes) then starting.

Reply:
Valid, that's not how I've ever written it at work so I assumed my way was the valid way. Either way works.